libraries/mbed/hal/can_api.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     IRQ_PASSIVE,
     IRQ_ARB,
     IRQ_BUS,
-    IRQ_READY
+    IRQ_READY,
+    IRQ_RX1
```

Comment:
Imagine how this would be used by the application? Yo ushould have a look at Sam's comment : https://github.com/mbedmicro/mbed/pull/1520#issuecomment-183126746 . How an app register callbacks?